src/ophys_etl/transforms/convert_rois.py

```diff
@@ -147,12 +154,24 @@ def binarize_and_create(self):
             movie_shape = open_vid['data'][0].shape
 
         # binarize the masks
-        self.logger.info("Binarizing the ROIs created by Suite2p.")
+        self.logger.info("Filtering and Binarizing the ROIs created by "
+                         "Suite2p.")
         coo_rois = suite2p_rois_to_coo(suite2p_stats, movie_shape)
 
+        # filter raw rois by linear dimension
+        edge_trsh = self.args['longest_edge_threshold']
+        self.logger.info("Linearly filtering ROIs to remove suite2p "
+                         "border artifacts")
+        linear_filtered_rois = filter_rois_by_longest_edge_length(coo_rois,
+                                                                  edge_trsh)
+        self.logger.info("Filtered out "
+                         f"{len(coo_rois) - len(linear_filtered_rois)} "
+                         f"raw rois with linear dimension threshold of "
```

Comment:
A linear dimension is the sum of the different dimensions -- so, in this case height and width. But I'm pretty sure what you mean is that neither the height nor width may exceed this value (not that the sum can't exceed). Please correct.

Reply:
Remove linear dimension thershold for more accurate logging message